packages/entity-browser/src/main.js

```diff
@@ -28,10 +28,24 @@ const createHistory = store => createHashHistory({
   }
 })
 
+const initHistory = (store, input) => {
+  const history = createHistory(store)
```

Comment:
i personally would create the store outside of initHistory and pass it instead of store and name initHistory something like "navigate to detail if initial key set" to make intentions clearer